lua/plugins.lua

```diff
@@ -82,15 +82,15 @@ return require("packer").startup(function(use)
   use {
     "mhartington/formatter.nvim",
     config = function()
-      require "core.formatter"
+      require("core.formatter").generic_config()
```

Comment:
Please use `setup` instead of `generic_config` like for other plugins

Reply:
This one is tricky, because we're not actually setting it up just yet here. That is done per language. Then I also considered `setup_generic()` but it's not that much better either. 
Alternatively, I was considering brining back `autocmds.lua` or maybe it was never there (?)
What do you think?

Reply:
This is still a behavior that we set up only once the plugin is loaded, I would keep the same naming as IMO we don't need to differentiate them from other plugins.

Reply:
Also, the config callback is just a hook provided by packer so that additional behavior can be defined, which is exactly what we do by defining augroups